### ISSUE TITLE
Vignettes not opening in RStudio 2022.12.0.353, gives error "Access to 127.0.0.1 was denied"

### ISSUE THREAD

**Issue by cnchapman** [rstudio/rstudio#12714]
### System details

    RStudio Edition : Desktop
    RStudio Version : 2022.12.0.353
    OS Version      : Mac OS X 13.1 (Ventura)
    R Version       : 4.2.2

### Steps to reproduce the problem
Choose the "Packages" tab in RStudio
select "fortunes" package [for example; happens in multiple] 
select "User guides, vignettes ..."
click on "PDF" [for the packagevignette]
The following error displays in Chrome (version 110.0.5481.100 (Official Build) (arm64))

### Describe the problem in detail
Opening a vignette gives an HTTP 403 error, permission denied to open the PDF in browser (in this case, Chrome 110.0.5481.100 [latest]).

According to R-SIG-Mac, this may be related to the Electron framework adopted for RStudio 2022.12.0.353, see: https://stat.ethz.ch/pipermail/r-sig-mac/2023-February/014661.html  but there is no resolution yet. The solution reported there is to downgrade RStudio.

I additionally reinstalled, unlinked, and relinked R, but that had no apparent effect.

### Describe the behavior you expected
Open the vignette PDF in the browser

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@cnchapman Thank you for raising this! I can also reproduce this with `fortunes` and `survival` in RStudio Desktop 2023.03.0-daily+318 on MacOS 13.2.

Also note:
1. Using `vignette()` in the Console Pane works, e.g. `vignette("fortunes")` and `vignette("survival")`, which opens the vignette in an external app such as Adobe Acrobat Reader.
2. HTML works, e.g. with the `broom` package. 
3. This works in RStudio Server, as it might be an Electron-specific issue as suggested.

**Comment by LeonBarmuta** [comment]
I have the same issues and behaviour under  Windows 10 with RStudio 2023.06.0 (full version info: RStudio 2023.06.0+421 "Mountain Hydrangea" Release (583b465ecc45e60ee9de085148cd2f9741cc5214, 2023-06-05) for windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2023.06.0+421 Chrome/110.0.5481.208 Electron/23.3.0 Safari/537.36). Thoughts or workarounds other than typing 'vingnette' at the console?

**Comment by aphalo** [comment]
Under Windows 10 RStudio 2023.06.2 still has displaying of PDF vignettes broken, but VS Code opens them. The URLs used for package 'multcomp' are.
RStudio (access denied): http://127.0.0.1:41419/help/library/multcomp/doc/chfls1.pdf
VS Code (working): localhost:30260/library/multcomp/doc/chfls1.pdf

RStudio 2023.06.2+561 "Mountain Hydrangea" Release (de44a3118f7963972e24a78b7a1ad48b4be8a217, 2023-08-25) for windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) RStudio/2023.06.2+561 Chrome/110.0.5481.208 Electron/23.3.0 Safari/537.36

**Comment by kevinushey** [comment]
Internal note: the problem here is that we end up trying to access the RStudio proxy in front of the R help server from an external browser. When opening Help links in an external browser, we should instead try to access the R help server directly.

**Comment by kevinushey** [comment]
Seems like this has since been fixed; I see the PDF vignette opened with the system viewer (Preview in my case).

**Comment by cnchapman** [comment]
Agreed. It is working for me now also. Will close (for now at least).
